src/app/modules/main/chat_section/controller.nim

```diff
@@ -139,6 +144,9 @@ method isCommunity*(self: Controller): bool =
 method getJoinedCommunities*(self: Controller): seq[CommunityDto] =
   return self.communityService.getJoinedCommunities()
 
+method getCommunity*(self: Controller, communityId: string): CommunityDto =
+  return self.communityService.getCommunityById(communityId)
```

Comment:
I guess I wasn't clear. I meant we can get rid of the `communityId` altogether, since the chat_section represents only one community. In this case, `self.sectionId` is the `communityId`. 
From the way you use the function, it seems like you will never actually fecth the info of another community other than the one represented by `sectionId`. 
That was why I suggested changing it to `getCommunity`, or maybe `getMyCommunity` if it's clearer.

Reply:
aaah makes sense. My bad I didn't catch that